app/controllers/api-keys/post-revoke.controller.js

```diff
@@ -20,8 +21,9 @@ module.exports = (req, res) => {
       req.flash('generic', 'The API key was successfully revoked')
       res.redirect(paths.apiKeys.index)
     })
-    .catch(err => {
-      req.flash('genericError', `<h2>Something went wrong</h2><p>${err}</p>`)
+    .catch(error => {
+      logger.error('Error revoking API key', { error })
```

Comment:
As our log ingestion has a text length limit I wonder if it would be best to just log the message here. This shouldn't be an issue for now (especially if we know what kind of error we're catching) and will likely give other helpful info like a code but I think it would be good to be explicit about what the log should include to avoid overflowing the structured logging.

Reply:
good point, I'll change this